Clarification on kotlin-debug-adapter role in readme (#480)
fwcd/kotlin-language-server: README.md

* See [Troubleshooting](TROUBLESHOOTING.md) for tips on troubleshooting errors
* See [Roadmap](https://github.com/fwcd/kotlin-language-server/projects/1) for features, planned additions, bugfixes and changes
* See [Kotlin Quick Start](https://github.com/fwcd/kotlin-quick-start) for a sample project

> * See [Kotlin Debug Adapter](https://github.com/fwcd/kotlin-debug-adapter) for running and debugging support on the JVM inside your editor 

Reviewer: "...on JVM" in this context was short for "...on [the] JVM as a platform", i.e. to clarify that Kotlin/{JS,Native} isn't supported (yet). Given that the language server hasn't supported anything besides Kotlin/JVM this probably hasn't been a major point of confusion, though.

One could interpret this sentence as "...on the JVM inside your editor" which feels a bit specific to me. Perhaps

```suggestion
* See [Kotlin Debug Adapter](https://github.com/fwcd/kotlin-debug-adapter) for editor-agnostic launch and debug support of Kotlin/JVM programs
```

(Yeah, this is probably bikeshedding at this point. Merging the PR as-is would work for me too, just want to hear your thoughts on these.)

Author: Your suggestion works slightly better, so using that 😄
